src/main/java/org/kiwiproject/migrations/mongo/AbstractMongockCommand.java

```diff
@@ -29,18 +29,13 @@ protected void run(Bootstrap<T> bootstrap, Namespace namespace, T configuration)
         var mongoClient = MongoClients.create(migrationConfiguration.getMongoUri(configuration));
 
         var driver = MongoCore3Driver.withDefaultLock(mongoClient, migrationConfiguration.getDatabaseName(configuration));
```

Comment:
Oh, now I see what you meant about potentially making the Mongo driver version configurable. I did not realize it would be this easy - at least it seems like it would only change this line (plus an additional configuration parameter).